sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java

```diff
@@ -633,6 +640,10 @@ protected ScheduleCreateTransaction doSchedule(TransactionBody.Builder bodyBuild
         return scheduled;
     }
 
+    protected boolean isBatchedAndNotBatchTransaction() {
+        return batchKey != null && !(this instanceof BatchTransaction);
```

Comment:
are we keeping this check - `!(this instanceof BatchTransaction)`  just for better readability of the code? 

I think we already override it in the batchtransaction to `return true`.